src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -30,7 +30,7 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
 
   "A WorkflowManagerActor" should {
 
-    val TestExecutionTimeout = 5000 milliseconds
+    val TestExecutionTimeout = 5 seconds
```

Comment:
Whoops I hadn't realized I needed to do `.dilated` to get that behavior.  :frowning: